CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+
+#set version of cmake to version on server
+cmake_minimum_required(VERSION "3.18.4")
```

Comment:
"If the running version of CMake is lower than the <min> required version it will stop processing the project and report an error. " -cmake_minimum_required() page on cmake.org.

Or is there a deeper reason?